Dockerfile

```diff
@@ -1,16 +1,29 @@
 # vim:set ft=dockerfile:
-FROM vertexproject/synapse-base-image:py36
 
-ENV SYN_DMON_LOG_LEVEL="WARNING"
```

Comment:
Don't we still use this to control logging in dmon? is there a reason to remove its default value?

Reply:
I'll add this back